sharness/t0030-ctl-pin.sh

```diff
@@ -88,6 +88,14 @@ test_expect_success IPFS,CLUSTER "pin data to cluster with user allocations" '
     echo $allocations | grep -q ${pid}
 '
 
+test_expect_success IPFS,CLUSTER "pin update a pin" '
+   cid1=`docker exec ipfs sh -c "echo test | ipfs add -q"`
+   ipfs-cluster-ctl pin add "$cid1"
```

Comment:
We can't replace these two lines with `echo test | ipfs-cluster-ctl add -q`.  Would it be helpful to have that someday?

Reply:
Yes, it would be useful. New feature request?